GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -327,3 +323,436 @@ def browserCONF(self):
+                        HEY.setWindowIcon(QtGui.QIcon(':/Pictures/images/HEY.png'))
+                        HEY.exec_()
+                    elif not numberBOOL:
+                        HEY = QMessageBox()
+                        HEY.setWindowTitle('Hey! Listen!')
+                        HEY.setText("Hey! Your password must have at least 1 number! [Ex: !@#$%^%&]")
+                        HEY.setIcon(QMessageBox.Critical)
+                        HEY.setWindowIcon(QtGui.QIcon(':/Pictures/images/HEY.png'))
+                        HEY.exec_()
+                    else:
+                        if self.adminyn == 'y':
+                            print('This user will be an admin')
+                            print(os.getcwd())
+                            command = """
+$nusnm = """ + '"{}"'.format(username) + """
+$nuspss = ConvertTo-SecureString """ + '"{}"'.format(passwd) + """ -AsPlainText -Force
+New-LocalUser -Name $nusnm -Password $nuspss
+Add-LocalGroupMember -Group "Administrators" -Member $nusnm"""
+                            print(command)
+                            sub.Popen(["powershell", "& {" + command + "}"])
+                            print('User ' + username + " has been successfully added!")
+                            completedPOP()
+                        elif self.adminyn == 'n':
+                            print('This user will not be an admin')
+                            command = """
+$nusnm = """ + '"{}"'.format(username) + """
+$nuspss = ConvertTo-SecureString """ + '"{}"'.format(passwd) + """ -AsPlainText -Force
+New-LocalUser -Name $nusnm -Password $nuspss"""
+                            print(command)
+                            sub.Popen(["powershell", "& {" + command + "}"])
+                            print('User ' + username + " has been successfully added!")
+                            completedPOP()
+
+                def cancelbutton():
+                    self.close()
+
+                self.Confirm_button.clicked.connect(lambda: threader(CONFIRM()))
+                self.Cancel_button.clicked.connect(cancelbutton)
+
+                def threader(com):
+                    try:
+                        threader = Thread(target=com)
+                        threader.start()
+                    except Exception as e:
+                        print(e)
+                        print('Could not start thread')
+
+        def calladdusrtosys():
+            widget = addusrtosys()
+            widget.exec_()
+
+        calladdusrtosys()
+
+    def remusr(self):
+        pass
+
+    def addgrutosys(self):
+        pass
+
+    def remgrufrosys(self):
+        pass
+
+    def addusrtogru(self):
+        pass
+
+    def remusrfrogru(self):
+        pass
+
+    def lslocausrs(self):
+        pass
+
+    def lslocagrus(self):
+        pass
+
+    def lsmemofgru(self):
+        pass
+
+    def lsgrusanusrin(self):
+        pass
+
+    # @staticmethod
+    def chngpasswdofall(self):
```

Comment:
Codacy found an issue: [funcWINusrgru.chngpasswdofall is too complex (41) (MC0001)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5823742)